wazo_calld/plugins/endpoints/services.py

```diff
@@ -248,7 +248,12 @@ def _update_trunk_cache(self, trunks):
             if trunk.get('endpoint_sip'):
                 techno = 'sip'
                 name = trunk['endpoint_sip']['name']
-                username = trunk['endpoint_sip']['username']
+                for key, value in trunk['endpoint_sip'].get('auth_section_options', []):
```

Comment:
auth_section_options is part of the spec
```suggestion
                for key, value in trunk['endpoint_sip']['auth_section_options']:
```